easycv/apis/export.py

```diff
@@ -655,23 +616,27 @@ def trace_module(self, **kwargs):
         self.model = trace_model
 
     def forward(self, image):
-        preprocess_outputs = ()
 
         with torch.no_grad():
-            if self.preprocess_fn is not None:
-                output = self.preprocess_fn(image)
-                # if multi values ​​are returned, the first one must be image, others ​​are optional,
-                # and others will all be passed into postprocess_fn
-                if isinstance(output, tuple):
-                    image = output[0]
-                    preprocess_outputs = output[1:]
-                else:
-                    image = output
-
             model_output = self.model.forward_export(image)
 
-            if self.postprocess_fn is not None:
-                model_output = self.postprocess_fn(model_output,
-                                                   *preprocess_outputs)
-
         return model_output
+
+
+class ProcessExportWrapper(torch.nn.Module):
```

Comment:
named with ProcessExportWrapper, but it seems has nothing to do with export

Reply:
we use the wrapper to trace the model and export a preprocess jit model.